docker/templates/new-django-app/{{cookiecutter.app_name}}/package.json

```diff
@@ -10,6 +10,7 @@
     "babel-preset-env": "^1.7.0",
     "babelify": "^10.0.0",
     "browserify": "^16.5.0",
+    "core-js": "^3.8.3",
```

Comment:
I'm trying to retroactively add this babel setup to an application. Do I need this package?

Reply:
@smcalilly Yes, I believe you'll need it [so Babel can polyfill features correctly](https://babeljs.io/docs/en/babel-preset-env#usebuiltins).